File: src/main/java/com/ballgame/StaticImage.java
package com.ballgame;

import java.awt.Graphics2D;
import java.awt.image.*;
import java.util.*;

public class StaticImage extends GameObject {
    private int width;
    private int height;
    
    public void update(double deltaTime, double previousTime){
        this.incrementImageFrame(deltaTime, previousTime);
    }

    public void render(Graphics2D g2d){
     g2d.drawImage(this.image, (int) this.x, (int) this.y , this.width, this.height, null);
    }

    public int getWidth(){ return this.width; };

    public int getHeight(){ return this.height; };

    public StaticImage(int x, int y, int width, int height, ArrayList<BufferedImage> sprites){
        super(x,y);
        this.width = width;
        this.height = height;
        this.initSprites(sprites, 1, 0d);

    }
}

File: src/main/kotlin/com/ballgame/StaticImage.kt
package com.ballgame

import java.awt.Graphics2D
import java.awt.image.BufferedImage

class StaticImage(x: Int, y: Int, val width: Int, val height: Int, sprites: ArrayList<BufferedImage>) :
	GameObject(x, y) {

	override fun update(deltaTime: Double, previousTime: Double) {
		incrementImageFrame(deltaTime, previousTime)
	}

	override fun render(g2d: Graphics2D?) {
		g2d!!.drawImage(image, x.toInt(), y.toInt(), width, height, null)
	}

	init {
		initSprites(sprites, 1, 0.0)
	}
}
